scripts/dream.py

```diff
@@ -508,6 +509,8 @@ def create_cmd_parser():
         action='store_true',
         help='skip subprompt weight normalization',
     )
+    parser.add_argument('-sampler', '--user_sampler', default=None, type=str, choices=['ddim', 'k_dpm_2_a', 'k_dpm_2', 'k_euler_a', 'k_euler', 'k_heun', 'k_lms', 'plms'],
```

Comment:
@blessedcoolant; Total nitpick, but will you reformat this to look like the other add_args? You could also just run 'blue' on it to fix it.

Reply:
Done.